RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts+View.swift

```diff
@@ -2,14 +2,15 @@ import ComposableArchitecture
 import SwiftUI
 extension ImportMnemonicControllingAccounts.State {
 	var viewState: ImportMnemonicControllingAccounts.ViewState {
-		.init(isSkippable: entitiesControlledByFactorSource.isSkippable)
+		.init(isSkippable: entitiesControlledByFactorSource.isSkippable, disableSkipImport: disableSkipImport)
 	}
 }
 
 // MARK: - ImportMnemonicControllingAccounts.View
 extension ImportMnemonicControllingAccounts {
 	public struct ViewState: Equatable {
 		let isSkippable: Bool
+		let disableSkipImport: Bool
```

Comment:
why is this needed? why is not `isSkippable` enough?

Reply:
leftover, will remove